Source/Core/DolphinQt/NetPlay/NetPlayDialog.cpp

```diff
@@ -1089,6 +1117,8 @@ void NetPlayDialog::LoadSettings()
   const bool golf_mode_overlay = Config::Get(Config::NETPLAY_GOLF_MODE_OVERLAY);
 
   m_buffer_size_box->setValue(buffer_size);
+  m_buffer_size_box->setEnabled(!auto_buffer);
```

Comment:
This doesn't make sense to have here because the state depends on session-specific variables. The other call I reviewed should take care of it.